companies/volkswagen-ag.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "volkswagen-ag",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Volkswagen AG",
+    "address": "Volkswagen AG\nPrivacy Team\nBerliner Ring 2\n38440 Wolfsburg\nGermany",
+    "email": "privacy@volkswagen.de",
```

Comment:
[This page](https://datenschutz.volkswagen.de/kunden) also lists a phone number at the bottom. Please also add that.

Reply:
I tried adding all the numbers one by one, but they weren't passing the validation on https://company-json.netlify.app/ - so I removed it. Should I add them anyway manually?

Reply:
The numbers need to be formatted correctly to pass the validation. The `f` button in the generator can be used to format them.

Interestingly, the formatting library fails for the 0800 number. The correct format would be: `+49 800 8932836724889`

Reply:
Thanks! Have added it as per your suggestion.